custom_components/eufy_security/model.py

```diff
@@ -43,7 +43,7 @@ class Config:
     host: str = ConfigField.host.value
     port: int = ConfigField.port.value
     sync_interval: int = ConfigField.sync_interval.value
-    rtsp_server_address: str = ConfigField.host.value
```

Comment:
I did this intentionally assuming that RTSP server address would be identical host address, by default.

If you look at `parse` , we are reading from actual configuration